SofaKernel/modules/SofaBaseVisual/VisualModelImpl.cpp

```diff
@@ -554,14 +560,14 @@ bool VisualModelImpl::load(const std::string& filename, const std::string& loade
         }
         else
         {
-            serr << "Mesh \"" << filename << "\" not found" << sendl;
+            msg_error() << "Mesh \"" << filename << "\" not found";
         }
     }
     else
     {
         if ((m_positions.getValue()).size() == 0 && (m_vertices2.getValue()).size() == 0)
         {
-            sout << "VisualModel: will use Topology." << sendl;
+            msg_info() << "VisualModel: will use Topology.";
```

Comment:
No need to prefix "VisualModel:" as it the emitter is part of the msg_info in a normalized way.